Sources/SotoCore/RetryPolicy.swift

```diff
@@ -93,10 +93,10 @@ extension StandardRetryPolicy {
                 }
             }
             return .dontRetry
+#if DEBUG
```

Comment:
Is there a reason this is wrapped in `#DEBUG`?

Reply:
Commonly when debugging AWS would close the connection because the client didn't respond quick enough. To avoid things failing I added the retry on connect. But you can't have this in a release situation because when you receive a connection close error you don't know if the request got through, you just didn't receive a response. If you retry you could end up running the operation twice. This is a problem if the operation is non-idempotent. @fabianfett was one to point this out to me.